src/pretalx/orga/templates/orga/cfp/question_detail.html

```diff
@@ -13,6 +13,18 @@ <h4>
     </a>
 </h4>
 
+{% if question.active %}
+  <div class="alert alert-info">
+    {% trans "This question is currently publicly visible." %}
```

Comment:
Hmm, it might be better to explain what this means. Maybe something like "This question is currently active, it will be asked during submission." and "This question is currently inactive, and will not be asked during submission."